Controller/OneClick/PlaceOrder.php

```diff
@@ -192,6 +192,11 @@ public function execute()
         $resultJson = $this->resultJsonFactory->create();
 
         try {
+            $giftCardDiscountAmount = 0;
+            //Check if the customer has applied GiftCards
+            if($quote->getMageworxGiftcardsAmount()) {
+                $giftCardDiscountAmount = abs($quote->getMageworxGiftcardsAmount());
+            }
```

Comment:
This can be simplified with:
```suggestion
            // Check if the customer has applied GiftCards
            $giftCardDiscountAmount = !empty($quote->getMageworxGiftcardsAmount())
                ? abs($quote->getMageworxGiftcardsAmount())
                : 0;
```